src/main/java/gov/nasa/cumulus/metadata/aggregator/processor/FootprintProcessor.java

```diff
@@ -210,14 +206,64 @@ public String geometryToUMMG(Geometry geometry, String cmrStr) {
         if (lineTypes.size() > 0) geometryType.setLines(lineTypes);
         if (gPolygonTypes.size() > 0) geometryType.setGPolygons(gPolygonTypes);
 
+        spatialExtentType = removeBBX(spatialExtentType);
         String spatialExtentStr = gsonBuilder.toJson(spatialExtentType);
+        //TODO : This area actually output both footprint/GPolygon and BoundingRectangle
```

Comment:
What's this TODO? Does it still need to be addressed or can we remove this comment?

Reply:
good catch, Stepheny.  Modified